src/index.ts

```diff
@@ -75,6 +75,10 @@ export default class extends Controller {
       .then((html: string) => {
         this.element.innerHTML = html
 
+        this.element.dispatchEvent(
```

Comment:
What about using the built in dispatch mechanism? https://stimulus.hotwired.dev/reference/controllers#cross-controller-coordination-with-events

Reply:
Yes i can swap to this

Reply:
Thanks!

Reply:
I think we can also add an event when fetch starts and when it fails as well

What do you think?

Reply:
Yes as asked in the PR desc, it could be great
Will see to add such event, using the stimulus dispatch